.eslintrc.js

```diff
@@ -10,6 +10,8 @@ module.exports = {
   extends: [
     './packages/eslint-config-fb-strict/index.js',
     'plugin:import/errors',
+    'prettier',
+    'prettier/flowtype',
```

Comment:
Doesn't seem like any flowtype style rules were applied, so might want to remove this